scripts/globals/spells/damage_spell.lua

```diff
@@ -710,6 +710,21 @@ xi.spells.damage.calculateDivineEmblemMultiplier = function(caster, skillType)
     return divineEmblemMultiplier
 end
 
+-- Elemental seal applies its own multiplier to spells when Laevateinn is equipped,
+-- or some other source of ENHANCES_ELEMENTAL_SEAL is available to the caster.
+xi.spells.damage.calculateEleSealMultiplier = function(caster, skillType)
+    local eleSealMultiplier = 1
+
+    if
+        caster:hasStatusEffect(xi.effect.ELEMENTAL_SEAL) and
+        skillType == xi.skill.ELEMENTAL_MAGIC
+    then
+        eleSealMultiplier = 1 + caster:getMod(xi.mod.ENHANCES_ELEMENTAL_SEAL) / 100
+    end
```

Comment:
I would assume this does not boost non-elemental spells. Feed the action element to the function.
```lua
    if
        caster:hasStatusEffect(xi.effect.ELEMENTAL_SEAL) and
        spellElement >= xi.element.FIRE -- TODO: Test impact and meteor.
        skillType == xi.skill.ELEMENTAL_MAGIC
    then
        eleSealMultiplier = 1 + caster:getMod(xi.mod.ENHANCES_ELEMENTAL_SEAL) / 100
    end
```